src/utils.ts

```diff
@@ -162,7 +162,8 @@ export const getShortcutKey = (shortcut: string): string => {
       .replace("Alt+", "⌥")
       .replace("Ctrl+", "⌃")
       .replace("Shift+", "⇧")
-      .replace("Del", "⌫")}`;
+      .replace("Del", "⌫")
```

Comment:
Btw, I didn't catch this until now, but is this a unicode sign for `Backspace`?

Reply:
Checking Apple store pics, it looks like they use ⌦ for regular delete.

Btw I'm curious, is the `delete` key above `|` actually a `backspace` (as it should be)?

![image](https://user-images.githubusercontent.com/5153846/78933051-ec8d5080-7aa8-11ea-898e-7a5f44a8a46b.png)

Reply:
And I realized just now that we can delete elements both with backspace & delete. So you can keep it since Mac doesn't seem to differentiate between those, label-wise.

Merge whenever you feel like it.

Reply:
delete is the backspace on mac.. if you don't have the external keyboard.